src/components/Login.js

```diff
@@ -0,0 +1,105 @@
+import React,{useState, useContext} from 'react';
+import { NavLink, useHistory } from 'react-router-dom';
+import axios from 'axios';
+import decoration from '../assets/Decoration.svg';
+import Header from './Header';
+import { LoginContext } from './LoginContext';
+
+const Login=()=>{
+
+    const history = useHistory();
+    const [userEmail, setUserEmail] = useState('');
+    const [userPassword, setUserPassword] = useState('');
+    const [invalidMail, setInvalidMail] = useState('');
+    const [invalidPass,setInvalidPass] = useState('');
```

Comment:
Try to not use shortcuts like _pass_, naming that password is better for readability